README.md

```diff
@@ -44,62 +141,57 @@ On Ubuntu, an install script is available on each release [here](https://github.
 ./semgrep-v0.9.0-ubuntu-generic.sh
 ```
 
-To try `semgrep` without installation, you can also run it via [Docker](https://docs.docker.com/install/):
+To try Semgrep without installation, you can also run it via [Docker](https://docs.docker.com/install/):
 
 ```
 docker run --rm -v "${PWD}:/home/repo" returntocorp/semgrep --help
 ```
 
-## Usage
+## Terminology
 
-### Example Usage
+A **rule** (or *check*) is a Semgrep pattern that can be used to tell match
```

Comment:
Aha, see the earlier comment about rules vs checks. I don’t have a strong preference either way except that we remain consistent.

Reply:
“tell match source code” seems a bit clunky.